CONTRIBUTING.md

```diff
@@ -160,19 +220,70 @@ make html -b linkcheck
 
 The finished documentation can be found in the `docs/_build/html` directory.
 
-
-## Release Checklist
-
-The following is a checklist to complete before taging a release.
-This helps us ensure everything goes smoothely before our army of robots send
-PyVista out to PyPI and Conda-Forge for deployment. Please checkout the
-latest state of the `master` branch locally and:
-
-1. Run all tests as outlined in the [Testing Section](#testing) and ensure
-all are passing.
-
-2. Ensure all builds and tests on Azure are passing for the latest commits on
-the `master` branch
+#### Creating a New Pull Request
+
+Once you have tested your branch locally, push your branch to (pyvista
+GitHub)[https://github.com/pyvista/pyvista] and create a pull request
+while merging to master.  This will automatically run continious
+integration (CI) testing and verify your changes will work across
+several platforms.
+
+To ensure someone else reviews your code, at least one other member of
+the pyvista contributiors group must review and verify your code meets
+our community's standards.  Once approved, if you have write
+permission you may merge the branch.  If you don't have write
+permission, the reviewer or someone else with write permission will
+merge the branch and close out the PR.
+
+Since it may be necessary to merge your branch with the current
+release branch (see below), please do not close out your branch if it
+is a `fix/` branch.
+
+
+### Branching Model
+
+This project follows a modified
+[gitflow](https://datasift.github.io/gitflow/IntroducingGitFlow.html)
```

Comment:
I think the processes described aren't really that close to gitflow - we are squash merging and stable branches instead of a master branch that holds release but not development. I think oneflow would be closer, but still not really the same: https://www.endoflineblog.com/oneflow-a-git-branching-model-and-workflow. I'd personally just drop the references, the explanations here together with looking at what is actually happening on pyvista should be clear enough imo.

Reply:
Agreed.  Dropping links and just starting with the explanation.